### ISSUE TITLE
regions.Regions.read not correctly converting sexagesimal hours to decimal degrees

### ISSUE THREAD

**Issue by apmtowner** [astropy/regions#407]
I am using regions.Regions.read to read a large number of .crtf regions into astropy. These CRTF files have all been created interactively in CARTA. In most cases the coordinates of my region points/vertices are in decimal degrees, but in one case (outside of my control) the coordinates are in sexagesimal format - hours for RA, degrees for Dec. Regions seems to be correctly converting sexagesimal to decimal format, but it is not converting the RA value from hours to degrees. For instance, for a .crtf file with the content:
```
#CRTFv0 CASA Region Text Format version 0
symbol [[18:47:44.02103, -001.54.58.0916], .] coord=J2000, corr=[I], linewidth=2, linestyle=-, symsize=1, symthick=1, color=2ee6d6, font=Helvetica, fontsize=10, fontstyle=bold, usetex=false
symbol [[18:47:44.04110, -001.54.56.5869], .] coord=J2000, corr=[I], linewidth=2, linestyle=-, symsize=1, symthick=1, color=2ee6d6, font=Helvetica, fontsize=10, fontstyle=bold, usetex=false
symbol [[18:47:44.06118, -001.54.55.0070], .] coord=J2000, corr=[I], linewidth=2, linestyle=-, symsize=1, symthick=1, color=2ee6d6, font=Helvetica, fontsize=10, fontstyle=bold, usetex=false
symbol [[18:47:44.10133, -001.54.53.7280], .] coord=J2000, corr=[I], linewidth=2, linestyle=-, symsize=1, symthick=1, color=2ee6d6, font=Helvetica, fontsize=10, fontstyle=bold, usetex=false
```
I read this in with the command:
`reg = regions.Regions.read(filename.crtf,format='crtf')`
When I check the values of the coordinates in the region object, I get:
```
In [9]: reg
Out[9]:
<Regions([<PointSkyRegion(center=<SkyCoord (FK5: equinox=J2000.000): (ra, dec) in deg
    (18.7955614, -1.91613656)>)>, <PointSkyRegion(center=<SkyCoord (FK5: equinox=J2000.000): (ra, dec) in deg
    (18.79556697, -1.91571858)>)>, <PointSkyRegion(center=<SkyCoord (FK5: equinox=J2000.000): (ra, dec) in deg
    (18.79557255, -1.91527972)>)>, <PointSkyRegion(center=<SkyCoord (FK5: equinox=J2000.000): (ra, dec) in deg
    (18.7955837, -1.91492444)>)>])>
```
Despite what the output says above, the decimal values for RA have not, in fact, been converted to degrees (18h RA is ~270 degrees). There doesn't seem to be a way to tell Regions that the original RA value is in sexagesimal hours. The problem is also present when the region I read in is a polygonal region rather than a series of points (i.e. when Regions creates a PolygonSkyRegion instead of a series of PointSkyRegion objects).

I'm not sure if this is a bug or just a mode that is not yet supported. I would naively expect any RA value in sexagesimal format to be treated as being in hours automatically, which is why I think this may be a bug and am reporting it as an issue - if this is a mode that simply isn't supported yet, I can move this comment to the appropriate location.

**Comment by keflavich** [comment]
We've never seen this type of region before in the wild, and it's not part of the CARTA spec (https://casaguides.nrao.edu/index.php/CASA_Region_Format, https://casa.nrao.edu/casadocs/casa-5.4.0/image-analysis/region-file-format) afaict.  However, most likely, CARTA/CASA are using `:`s to indicate that the units are hours.  Fix forthcoming.

**Comment by apmtowner** [comment]
Thanks!
The region format ONLY occurs for one particular source, which was observed separately from the other sources in my sample. It's possible there's something specific to this data set that is producing this strange format. (I tried to force CARTA to output a different format with sky coordinates - as opposed to pixel coordinates - but could not succeed, which is part of why I think the issue might be something specific to these data.)

**Comment by larrybradley** [comment]
This was fixed in #408.
